ruby/basics/variables_strings_and_numbers.md

```diff
@@ -1,119 +1,371 @@
 
-* Learning outcome 1
-* Learning outcome 2
-* Learning outcome 3
 
 ## Variables
-You should know what variables are from the prep course and know how to assign values to a variable name
-.
+You can think of Variables as those tupperware containers your mum keeps in the cupboard (that never have matching lids...). You can use those containers to store something in them until you need whatever is in it. In programming a variable is simply a place to store data. To be more specific, variables hold a reference to a specific object in computer memory. Each object will refer to a certain data type.
 
-What you may not know are the common Ruby naming conventions for variables and some more advanced assignment operators so we'll cover them here.
+Although you should be familiar already with variables in Ruby what you may not know are the common Ruby naming conventions for variables and some more advanced assignment operators so we'll cover them here.
 
+### Declaring a Variable
+As you have already seen in the prep work this is how you make a variable in Ruby.
 
+```(ruby)
+age = 18 # => 18
+```
 
-### How to make them
-- assignment operators =, +=, -=, *=, /=, %=, **=
+You can also assign a variables value as the result of an operation
 
-Almost all programming languages use the `=` sign to assign values to variables.
+```(ruby)
+age = 18 + 5 # => 23
+```
 
+To change the value of a variable after its been already been assigned you will need to assign
+it again with the new desired value.
+
+```(ruby)
+age = 18
+age = 33
+age # => 33
 ```
-num = 5
+
+There will often be scenarios where you will want to do an operation to the original
+value of a variable and reassign the variable as the result of that operation. this
+is how you would do that:
+
+```(ruby)
+age = 18
+age = age + 4 # =>  22
 ```
-In the above statement, we're assigning the value `5` to the variable `num` using `=`.
 
-- parallel assignment
-We can also assign more than one variable in a single statement. Look at the following example.
+Because this is a common scenario Ruby provides a nice short hand
+assignment operator for doing this `+=`:
 
+```(ruby)
+age = 18
+age += 4 # => 22
 ```
-name, age = 'John', 34
+
+there are assignment operators like this for all the common math operators
+
+```(ruby)
+age = 18
+age -= 2 # => 16
+
+cash = 10
+cash *= 2 # => 20
+
+temperature = 40
+temperature /= 10 # => 4  
+```
+
+### How to name variables
+Ruby is a language which aims to be natural to read and easy to write. Remember this when you're naming your variables. The name should, as clearly as possible, aim to describe the what the value of the variable represents. Naming variables will pay dividends when you review your code months after you've written it when you can no longer clearly remember what that variable was designed to store. When deciding what your variable name will be keep this quote by John Woods at the front of your mind:
```

Comment:
Missing comma after 'what your variable name will be'

Can we simplify the last sentence? Something along the lines of

> From now on, when naming your variables, remember the following quote by John Woods

/cc @csrail @CouchofTomato @codyloyd